### ISSUE TITLE
Compute_20 nvcc error

### ISSUE THREAD

**Issue by fbcotter** [ContinuumIO/pyculib_sorting#8]
Downloading and trying to install gives me the following error. I think this is because I'm using CUDA 9, which may have remove support for fermi architectures.

    nvcc fatal   : Unsupported gpu architecture 'compute_20'

This was simple enough to fix by making the following change:

```
diff --git a/build_sorting_libs.py b/build_sorting_libs.py
index 1589b7e..d1f44f6 100644
--- a/build_sorting_libs.py
+++ b/build_sorting_libs.py
@@ -53,7 +53,7 @@ def gencode_flags():
 
     # Concatenate flags
     SM = []
-    SM.append(GENCODE_SM20)
+    #  SM.append(GENCODE_SM20)
     SM.append(GENCODE_SM30)
     SM.append(GENCODE_SM35)
     SM.append(GENCODE_SM37)
```

**Comment by dzhoshkun** [comment]
Indeed, [starting with 9.0, the minimum required CC is 3.0](https://stackoverflow.com/questions/28932864/cuda-compute-capability-requirements/28933055#28933055) (I couldn't find this information on the [CUDA documentation](https://docs.nvidia.com/cuda/) itself, but the link [was posted by an NVIDIA moderator](https://devtalk.nvidia.com/default/topic/1028960/cuda-setup-and-installation/looking-for-cuda-compatibility-chart-for-nvidia-drivers/)). So, it might be better to check the installed CUDA version, and append the architecture options accordingly.

**Comment by dzhoshkun** [comment]
@fbcotter I've submitted the above PR that should resolve this issue, while keeping backwards compatibility with older CUDA versions. It'd be great if you could give the changes a try on your platform as well.

**Comment by fbcotter** [comment]
@dzhoshkun I tried your commits on my system and it compiled and ran fine. Nice one.
